packages/canvas-renderer/src/CanvasRenderer.ts

```diff
@@ -227,20 +227,8 @@ export class CanvasRenderer extends SystemManager<CanvasRenderer> implements IRe
 
         // convert our big blob of options into system specific ones..
         const startupOptions: StartupOptions = {
-            hello: options.hello,
-            _plugin: CanvasRenderer.__plugins,
-            background: {
-                alpha: options.backgroundAlpha,
-                color: options.background ?? options.backgroundColor,
-                clearBeforeRender: options.clearBeforeRender,
-            },
-            _view: {
-                height: options.height,
-                width: options.width,
-                autoDensity: options.autoDensity,
-                resolution: options.resolution,
-                view: options.view,
-            }
+            plugins: CanvasRenderer.__plugins,
```

Comment:
Another idea:

If plugins was removed here we would not need StartupOptions type at all. This is kind of an outlier anyways. Maybe after this.setup call we manually share the reference to plugins:

```js
this._plugins.initPlugins = CanvasRenderer.__plugins;
```